### PR TITLE
Updating demo.css to only remove padding on sliders

### LINKED ISSUE #692: Input text is too close to edges
## First Time?

This is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/image-sequencer/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

## The problem

After updating some of our modules to have sliders instead of text input fields, some of the styling we provided for the sliders overwrote the styling for the text input fields as well. This resulted in all of our input fields having their padding removed.

For ex: [http://sequencer.publiclab.org/examples/#steps=crop{}](http://sequencer.publiclab.org/examples/#steps=crop{})

screen shot 2019-01-20 at 11 26 23 pm

## Solution

We will make our css declaration more specific so that it only removed padding on sliders:

On the demo.css page, for the below declaration add the attribute selector `[type="range"]` to the class selector `.form-control` so that it no longer effects the text input fields.

https://github.com/publiclab/image-sequencer/blob/17d79c4b44c5e5b8f7f8b4574a783d5b87cdf871/examples/demo.css#L45-L47

Thanks!!

## Steps to Fix
 
- [ ] Claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] Fork the repository and set it up locally following the main repo README instructions [https://github.com/publiclab/image-sequencer/blob/main/README.md](https://github.com/publiclab/image-sequencer/blob/main/README.md)
 - [ ] Create a new feature branch with a unique name descriptive to the issue
- [ ] Try to fix the issue following the steps above, but even before you're done, you can:
  commit your changes to your branch and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] Reference this issue in your pull request body
- [ ] Once you submit your pull request, an additional checklist will be provided for getting it merged

##  💬   Get help
If you need any help - here are some options: 

- Comment below
- Join our gitter chat at https://gitter.im/publiclab/publiclab

### PR DESCRIPTION
**Pull request by nic-og**
The demo.css was updated in order to add the attribute selector [type="range"] to the class selector `.form-control` so that it no longer affects the text input fields.

Fixes #692 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
@nic-og welcome and thanks for you PR!
Please attach a GIF/screenshot to the PR body. You can also mention @publiclab/is-reviewers in a PR comment to ask for a review.

File: examples/demo.css [hunk 1]
```diff
@@ -42,7 +42,7 @@ h1 {
   margin: 0 !important;
 }
 
-.form-control {
+.form-control [type="range"] {
```
Comment (HarshKhandeparkar) [inline examples/demo.css]: There should be no space between `.form-control` and `[type="range"]`

Comment (nic-og) [reply examples/demo.css]: Thanks for the review. I've committed the changes. 

space-removed

Comment (sashadev-sky) [reply examples/demo.css]: @HarshKhandeparkar I assigned myself to provide 2nd approval after you approve, so we can request a merge promptly :)

**Comment by HarshKhandeparkar** [comment]
@sashadev-sky you can add the ready-to-merge label after approving the changes. Thanks.

**Comment by sashadev-sky** [comment]
@nic-og Thank you for your code! The last thing we need before requesting merge is a screenshot of the results in your browser. In the future try to always provide these for frontend changes :)

**Comment by nic-og** [comment]
result

@HarshKhandeparkar  @sashadev-sky  Thank you for the help.

**Comment by sashadev-sky** [comment]
@nic-og Perfect, thank you!!

**Comment by jywarren** [comment]
Great teamwork, folks! Thanks so much!

**Pull request merged**